README.md

```diff
@@ -36,7 +36,7 @@ The table below describes all the possible configuration variables, as well as t
 | `repositories:{full_name}:gitHubToken` | `string` | Token used to verify __outgoing__ requests to GitHub repository | ✓ | 
 | `repositories:{full_name}:thirdPartyFolders` | `string` | Comma-separated list of folders in which to look for changed files in pull request to remind user to update License. | X | `[]`
 | `repositories:{full_name}:claUrl` | `string` | The GitHub API URL to the CLA file in JSON form. See [here](https://developer.github.com/v3/repos/contents/#get-contents) for what the URL should look like. _Example:_ https://api.github.com/repos/AnalyticalGraphicsInc/cesium-concierge/contents/specs/data/config/CLA.json | X | _Disabled if not set._
-| `repositories:{full_name}:contributorsUrl` | `string` |  The GitHub API URL to `CONTRIBUTORS.md`. | X | _Disabled if not set._
+| `repositories:{full_name}:contributorsPath` | `string` |  Relative path to the `CONTRIBUTORS.md` file. | X | _Disabled if not set._
```

Comment:
I know it's pretty obvious, but we should probably specify what it's relative to. Just here is fine.